include-build/roborazzi-core/build.gradle

```diff
@@ -46,7 +46,7 @@ kotlin {
     }
     jvmMain {
       dependencies {
-        implementation libs.json
+        implementation libs.gson
```

Comment:
We might need to add it to androidMain, as org.json is included in the Android Framework, whereas gson is not.

Reply:
Ok. I'll try it!